### ISSUE TITLE
Missing option for screen reader for present status

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#5571]
**Current behavior**
If a visually impaired user is present/not present that feedback is completely missing from the overlay

**Wanted behavior**
via screen reader the status if you are present/not present should be understtandable for you
